packages/fxa-settings/tailwind.config.js

```diff
@@ -63,6 +63,7 @@ module.exports = {
       black: '#000',
       white: '#fff',
       grey: {
+        20: '#FAFAFB',
```

Comment:
Oooh I totally need this in my PR, we might have a merge conflict depending on whose is merged first. :P

Reply:
How do you feel about 20? I used it somewhat arbitrarily.

Reply:
Changing to 10